pyamplipi/client.py

```diff
@@ -234,7 +234,7 @@ async def delete(self, path: str, body=None, headers=None) -> dict:
             async with self._http_session.delete(
                     url=self.url(path),
                     data=body,
-                    timeout=self._timeout,
+                    timeout=ClientTimeout(total=self._timeout),
```

Comment:
This param doesn't really want an int, but [this funny thing](https://docs.aiohttp.org/en/stable/client_quickstart.html#aiohttp-client-timeouts).